deps/include/brainflow/utils/brainflow_constants.h

```diff
@@ -51,6 +51,7 @@ enum class BoardIds : int
     IRONBCI_BOARD = 15,
     GFORCE_PRO_BOARD = 16,
     FREEEEG32_BOARD = 17,
+    BRAINALIVE_BOARD = 18,
```

Comment:
Ask @Andrey1994 for permission to add this to his repo https://github.com/brainflow-dev/brainflow otherwise this will be overwritten.

Reply:
Hi, I am glad to add new boards. Could you send a PR to brainflow upstream and I will review and merge it? After that we will be able to update that dependency in neuromore studio

Reply:
I reviewed that more carefully, it seems like you dont use brianflow api in this PR and didnt add new board to it.
In this case you dont need to change brainflow_constants .h and brainflow_boards.cpp, these files are only for devices supported by brainflow

Reply:
@dioptre 
Thank you for your response. I have read about brainflow, Indeed it's having prominent features which can be beneficial for us, we would certainly implement it in our future work after having ample understanding of it, but as of now, I would request you to merge this code in your main code after a peer review.
Along with this, I would like to thank Mr. Andrey Parfenov for his kind suggestions.

Reply:
@Sarthak031 you overwrote dependencies in the https://github.com/neuromore/studio/tree/master/deps brainflow files. You can see what you did here guys (mentioned you both) https://github.com/neuromore/studio/pull/102/commits/a8392d53e17007889b713d3c79561d9369ff5470#diff-42c9ab2453ad4ab9633ea9cc45948e03417de7bdc9bb429c9dd1e746b6a77623R275

Reply:
Not sure where it went?

Reply:
I think these changes are not needed. There are two steps to add new board to brainflow - add info about this board and register new board id(its done here) and the second one is to add board implementation to brainflow(inherit your object from abstract class and implement neeed methods, its not done in this PR).

And brainflow is not used here, neuromore driver for your new board communicates with device directly and doesnt use brainflow api.